### DISCUSSION TITLE
about the default method of interface hook

### DISCUSSION THREAD

**Discussion by buffcow** [LSPosed/LSPosed#2876] [Q&A] [answered]
as title, when i hook a default method

![image](https://github.com/LSPosed/LSPosed/assets/42611305/c5064452-70a6-4433-8551-37eb8b598eb0)

![image](https://github.com/LSPosed/LSPosed/assets/42611305/a14595c8-9a66-47dd-87f5-7c3f999b47d6)

It doesn't seem to work, is no log output either, even if it has been called
so Is my way wrong?

**Comment by JasonKhew96** [comment, marked as answer]
abstract class is not hookable
